app/javascript/app/acuant/document_capture.js

```diff
@@ -33,7 +50,7 @@ export const loadAndInitializeAcuantSdk = () => {
   window.onAcuantSdkLoaded = initializeAcuantSdk;
 
   const sdk = document.createElement('script');
-  sdk.src = 'AcuantJavascriptWebSdk.min.js';
+  sdk.src = 'AcuantJavascriptWebSdk.min.js?v=11.4.1';
```

Comment:
> We're already serving these out of Rails and not out of the static asset pipeline?

If the files are in `public`, I'm assuming Rails routing won't apply, and the files will be consumed as-is? Not sure if we'd necessary _want_ to continue keeping these files in this directory. Moving to assets could at least give us the fingerprinting for free.

As it stands, with versioning in the path, I'd imagined something like:

```
public/AcuantJavaScriptWebSdk-v11.4.1.min.js
```

Or:

```
public/acuant/11.4.1/AcuantJavaScriptWebSdk.min.js
```

(or other similar variations)

Reply:
We do have the files in the `public` directory but we serve them through Rails:

See https://github.com/18F/identity-idp/blob/master/config/routes.rb#L263-L267

Reply:
> We do have the files in the `public` directory but we serve them through Rails:
> 
> See https://github.com/18F/identity-idp/blob/master/config/routes.rb#L263-L267

Oh! I see. What you said originally makes sense now, and seems like an easy way to handle this.

I'll make those changes 👍

Reply:
So it seems like the easiest option would be:

```rb
AcuantSdkController::ACUANT_SDK_STATIC_FILES.each do |acuant_sdk_file|
  get "/verify/doc_auth(/:version)/#{acuant_sdk_file}" => 'acuant_sdk#show'
  get "/verify/capture_doc(/:version)/#{acuant_sdk_file}" => 'acuant_sdk#show'
  get "/verify/capture-doc(/:version)/#{acuant_sdk_file}" => 'acuant_sdk#show'
end
```

But this results in some strange URLs:

- `/verify/doc_auth/11-4-1/AcuantJavaScriptWebSdk.min.js`
   - Dashes instead of dots because Rails routing doesn't support dots in routes by default
   - `11-4-1` Isn't clear that it has anything to do with Acuant
- Or: `/verify/doc_auth/acuant-11-4-1/AcuantJavaScriptWebSdk.min.js`
- Or even: `/verify/doc_auth/nonsense/AcuantJavaScriptWebSdk.min.js`

Adding `constraints` here could help make this more rigid and appear nicer. And to do it for each, it might make sense to consolidate it to one route definition:

```rb
AcuantSdkController::ACUANT_SDK_STATIC_FILES.each do |acuant_sdk_file|
  get "/verify/:flow(/acuant/:version)/#{acuant_sdk_file}",
    to: 'acuant_sdk#show',
    constraints: {
      flow: /(doc_auth|capture[_-]doc)/,
      version: /\d+\.\d+\.\d+/
    }
end
```

URLs look nicer:

- `/verify/doc_auth/acuant/11.4.1/AcuantJavaScriptWebSdk.min.js`

... just not sure if that gets too far to the other end of being a bit magical for our tastes.

Reply:
Let's meet in the middle and [add the constraints via a block](https://guides.rubyonrails.org/routing.html#request-based-constraints)?

```ruby
constraints version: /\d+\.\d+\.\d+/ do
  get "/verify/doc_auth(/:version)/#{acuant_sdk_file}" => 'acuant_sdk#show'
  get "/verify/capture_doc(/:version)/#{acuant_sdk_file}" => 'acuant_sdk#show'
  get "/verify/capture-doc(/:version)/#{acuant_sdk_file}" => 'acuant_sdk#show'
end
```

Reply:
> Let's meet in the middle and [add the constraints via a block](https://guides.rubyonrails.org/routing.html#request-based-constraints)?

Looks good! Updated in 50a65d0c120f75410f3dcbef2980f6c6450ab176.